### ISSUE TITLE
while freeze the stylegan?

### ISSUE THREAD

**Issue by zhangyunming** [TencentARC/GFPGAN#8]
hi  nice work !

one question: why set  fix_decoder: true.  will fintune the param have a better result?

**Comment by xinntao** [comment]
As this work focuses on using the pre-trained StyleGAN, so we freeze it. 
We can also finetune it (may with a smaller learning rate). It probably improves a bit.
